src/libraries/Microsoft.PowerFx.Core/Public/Config/Features.cs

```diff
@@ -2,77 +2,92 @@
 
         /// <summary>
         /// Disables support for row-scope disambiguation syntax.
         /// Now,for example user would need to use Filter(A, ThisRecord.Value = 2) or Filter(A As Foo, Foo.Value = 2)
         /// instead of
-        /// Filter(A, A[@Value] = 2)
+        /// Filter(A, A[@Value] = 2).
         /// </summary>
-        DisableRowScopeDisambiguationSyntax = 0x4,
+        internal bool DisableRowScopeDisambiguationSyntax { get; set; }
 
         /// <summary>
-        /// Enable Identifier support for describing column names
-        /// Added on 6th December 2022.
+        /// Enable Identifier support for describing column names.
         /// </summary>
-        SupportColumnNamesAsIdentifiers = 0x8,        
+        internal bool SupportColumnNamesAsIdentifiers { get; set; }        
 
         /// <summary>
         /// Enforces strong-typing for builtin enums, rather than treating
-        /// them as aliases for values of string/number/boolean types
-        /// Added March 2023.
+        /// them as aliases for values of string/number/boolean types.
         /// </summary>
-        StronglyTypedBuiltinEnums = 0x10,
+        internal bool StronglyTypedBuiltinEnums { get; set; }
 
         /// <summary>
         /// Updates the IsEmpty function to only allow table arguments, since it
         /// does not work properly with other types of arguments.
         /// </summary>
-        RestrictedIsEmptyArguments = 0x20,
+        internal bool RestrictedIsEmptyArguments { get; set; }
 
         /// <summary>
         /// Allow delegation for async calls (delegate using awaited call result).
-        /// Added March 2023.
         /// </summary>
-        AllowAsyncDelegation = 0x40,
+        internal bool AllowAsyncDelegation { get; set; }
 
         /// <summary>
         /// Allow delegation for impure nodes.
-        /// Added March 2023.
         /// </summary>
-        AllowImpureNodeDelegation = 0x80,
+        internal bool AllowImpureNodeDelegation { get; set; }
 
         /// <summary>
         /// Updates the FirstN/LastN functions to require a second argument, instead of
         /// defaulting to 1.
         /// </summary>
-        FirstLastNRequiresSecondArguments = 0x100,
+        internal bool FirstLastNRequiresSecondArguments { get; set; }
+
+        internal bool PowerFxV1CompatibilityRules { get; set; }
+
+        internal static Features None => new Features();
 
-        /// <summary>        
+        public static Features PowerFxV1 => new Features
+        {
+            TableSyntaxDoesntWrapRecords = true,
+            ConsistentOneColumnTableResult = true,
+            DisableRowScopeDisambiguationSyntax = true,
+            SupportColumnNamesAsIdentifiers = true,
+            StronglyTypedBuiltinEnums = true,
+            RestrictedIsEmptyArguments = true,
+            AllowAsyncDelegation = true,
+            AllowImpureNodeDelegation = true,
```

Comment:
These two shouldn't be part of FxV1 but these are separate features and not done as part of V1 as such. In another words, these can be shipped after Fx 1.0 as well.

Reply:
Updated